bundles/org.openhab.binding.nuvo/src/main/java/org/openhab/binding/nuvo/internal/handler/NuvoHandler.java

```diff
@@ -1229,13 +1236,25 @@ private void cancelClockSyncJob() {
     }
 
     /**
-     * Update the state of a channel
+     * Update the state of a channel (original method signature)
      *
      * @param target the channel group
      * @param channelType the channel group item
      * @param value the value to be updated
      */
     private void updateChannelState(NuvoEnum target, String channelType, String value) {
+        updateChannelState(target, channelType, value, NO_ART);
+    }
+
+    /**
+     * Update the state of a channel (overloaded method to handle album_art channel)
+     *
+     * @param target the channel group
+     * @param channelType the channel group item
+     * @param value the value to be updated
+     * @param bytes the byte[] to load into the Image channel
+     */
+    private void updateChannelState(NuvoEnum target, String channelType, String value, byte[] bytes) {
```

Comment:
I think it will be cleaner to have a nullable final parameter (`bytes`), rather than using NO_ART with a comparison `NO_ART == bytes`.
`byte[] @Nullable bytes`

Reply:
I tried doing this but ran into this:
>  The nullness annotation @Nullable is not applicable for the primitive type byte
I have refactored to remove the comparison. The original path to setting the channel to UNDEF will be used to clear the image.